components/argocd/apps/overlays/rhoai-eus-2.8-aws-gpu/patch-operators-list.yaml

```diff
@@ -11,16 +11,6 @@ spec:
         values:
           name: nvidia-gpu-operator
           path: components/operators/gpu-operator-certified/aggregate/overlays/aws
-      - cluster: local
-        url: https://kubernetes.default.svc
-        values:
-          name: jaeger-operator
-          path: components/operators/jaeger/operator/overlays/stable
-      - cluster: local
-        url: https://kubernetes.default.svc
-        values:
-          name: kiali-operator
-          path: components/operators/kiali/operator/overlays/stable
```

Comment:
I made updates to remove the operators that aren't required from all of the different cluster options.